spring-cloud-stream-binders/spring-cloud-stream-binder-redis/pom.xml

```diff
@@ -44,6 +44,11 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-cloudfoundry-connector</artifactId>
```

Comment:
I believe this cf-connector dependency needs to be added to rabbit binder (not sure if we have kafka as cf service) as well. May be, this can be moved up to the binder parent?